app/browser/tabs.js

```diff
@@ -1015,6 +1017,11 @@ const api = {
           createProperties.isolated_storage = true
           createProperties.parent_partition = ''
           createProperties.tor_proxy = 'socks5://127.0.0.1:9050'
+          if (process.platform === 'win32') {
+            createProperties.tor_path = path.join(getExtensionsPath('bin'), 'tor.exe').quote()
```

Comment:
why is `quote` needed? `String.prototype.quote` does not appear to be supported on my version of Node (on MacOS)

Reply:
```
#if defined(OS_WIN)
// Windows-specific LaunchProcess that takes the command line as a
// string.  Useful for situations where you need to control the
// command line arguments directly, but prefer the CommandLine version
// if launching Chrome itself.
//
// The first command line argument should be the path to the process,
// and don't forget to quote it.
//
// Example (including literal quotes)
//  cmdline = "c:\windows\explorer.exe" -foo "c:\bar\"
BASE_EXPORT Process LaunchProcess(const string16& cmdline,
                                  const LaunchOptions& options);
```

Reply:
i see - did someone verify on windows that `string.quote()` works? it's non-standard https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/quote